File: test/fixtures/java-maven/src/main/java/com/example/domain/order/Order.java
package com.example.domain.order;

import java.util.List;
import java.util.ArrayList;

public class Order {
    private List<OrderItem> items;

    public Order() {
        this.items = new ArrayList<>();
    }

    public void addItem(OrderItem item) {
        this.items.add(item);
    }

    public List<OrderItem> getItems() {
        return items;
    }
}

File: test/fixtures/swift-project/Sources/Domain/Order.swift
import Foundation

/// Order domain entity
public struct Order {
    public let id: String
    public let items: [OrderItem]
    public var status: OrderStatus

    public init(id: String, items: [OrderItem], status: OrderStatus = .pending) {
        self.id = id
        self.items = items
        self.status = status
    }
}

/// Order item within an order
public struct OrderItem {
    public let productId: String
    public let quantity: Int
}

/// Order lifecycle status
public enum OrderStatus: String {
    case pending
    case confirmed
    case shipped
    case delivered
    case cancelled
}
